components/builder-depot/src/config.rs

```diff
@@ -101,11 +101,13 @@ impl GatewayCfg for Config {
         self.routers.as_slice()
     }
 }
+// maybe lowercase this shit
 
-#[derive(Debug, Clone, Deserialize, PartialEq)]
+#[derive(Debug, Clone, Deserialize)]
+#[allow(non_camel_case_types)]
 pub enum S3Backend {
-    Aws,
-    Minio,
+    aws,
```

Comment:
I'm not sure lowercasing the enum itself is the way to go.. all our enums are mixed case.  We may need to add a FromStr somewhere instead?

Reply:
I'm not sure exactly how that would work, but let me take a look and see if we can.